Brain/brain_utils.h

```diff
@@ -126,12 +123,24 @@ void take_action(valveInfo *valve, int action) {
   }
 }
 
+/*
+ * Calls the corresponding method for this valve with the appropriate 
+ * action in solenoids.h
+ */
+void take_action(valveInfo *valve, int action) {
+  if (action) {
+    valve->openValve();
+  } else {
+    valve->closeValve();
+  }
+}
+
```

Comment:
wtf. was this not in already
:P my bad.

Reply:
Haha no worries :)